Wikipedia/Code/WMFRelatedSectionBlackList.m

```diff
@@ -51,70 +60,20 @@ + (instancetype)loadFromDisk {
 
 - (void)addBlackListTitle:(MWKTitle*)title {
```

Comment:
that said, I thought we had talked about removing the locking mechanisms since this class is only going to be mutated on the main thread anyway. i'm ambivalent about whether the locking stays or goes, but either way I think it's better if the API surface area is kept to a minimum